main.nf

```diff
@@ -116,8 +116,8 @@ workflow {
 
         REPORTS_WF( QUALITY_CHECK_WF.out.reports_fastqc_ch,
                     UTILS_MERGE_COHORT_STATS.out.merged_cohort_stats_ch,
-                    MINOR_VARIANTS_ANALYSIS_WF.out.minor_variants_results_ch,
                     MERGE_WF.out.major_variants_results_ch,
+                    MINOR_VARIANTS_ANALYSIS_WF.out.minor_variants_results_ch,
```

Comment:
Beyond the tweak the `summarize_resistance` script, the order was reversed for passing the `major` and `minor` variants 😓